### DISCUSSION TITLE
Refiners - Greyed (No values for this filter)

### DISCUSSION THREAD

**Discussion by dvsRCalejo** [microsoft-search/pnp-modern-search#1421] [Q&A]
Hey guys,

I'm having this issue, that I don't know if it is a mistake I'm doing or not, but basically the filters webpart get greyed everytime I do a search on the search box. By default, it works ok, but everytime I do a search the filters get greyed.

Any thoughts?

![image](https://user-images.githubusercontent.com/11666192/137986082-7cd5306b-7562-40a7-b980-6e278bba47d0.png)

Thank you.

Ricardo Calejo

**Comment by wobba** [comment]
Have you followed the tutorial to connect the web parts and ensure you are using refinable managed properties. https://microsoft-search.github.io/pnp-modern-search/scenarios/page-with-filters/

**Reply by dvsRCalejo** [reply]
Yes. Everything is fine. I've looked for the reason on the Dev Tools, and I've found that for some searches, the refinement results doesn't bring any value. Can't understand why is this happening but it doesn't happen for all results.

Can you point me in the right direction?

**Reply by wobba** [reply]
Not without knowing what you are using in terms of managed properties and your cp mapping to them - and at what levels etc.
